src/app/modules/item/services/item-task-answer.service.ts

```diff
@@ -99,9 +112,6 @@ export class ItemTaskAnswerService implements OnDestroy {
   }
 
   reloadAnswer(): Observable<unknown> {
```

Comment:
If it clears the answer, `reloadAnswer()` is not well chosen as a name.

Reply:
Fixed